test/service/test_response.py

```diff
@@ -1451,6 +1454,29 @@ def test_disease_facet(self):
                 for facet_name, facet_value in facet_data.items():
                     self.assertEqual(facets[facet_name]['terms'], facet_value)
 
+    def test_organism_age_facet(self):
+        """
+        Verify the values of organism age facets
+        """
+        url = self.base_url + "/index/projects"
+        test_data = {
+            # organism_age specified in donor organism
+            '627cb0ba-b8a1-405a-b58f-0add82c3d635': {
+                'organismAge': [{'term': {'value': '20', 'unit': 'year'}, 'count': 1}],
+                'organismAgeUnit': [{'term': 'year', 'count': 1}],
+                'organismAgeValue': [{'term': '20', 'count': 1}],
+
+            },
+            # disease specified in donor only
+            '2c4724a4-7252-409e-b008-ff5c127c7e89': {
+                'organismAge': [{'term': {'value': '40-44', 'unit': 'year'}, 'count': 1},
+                                {'term': {'value': '55-59', 'unit': 'year'}, 'count': 1}],
+                'organismAgeUnit': [{'term': 'year', 'count': 1}],
+                'organismAgeValue': [{'term': '40-44', 'count': 1}, {'term': '55-59', 'count': 1}]
+            }
```

Comment:
```suggestion
            },
            # project has 1 donor organism but no organism_age properties
            'c765e3f9-7cfc-4501-8832-79e5f7abd321': {
                'organismAge': [{'term': {'value': None, 'unit': None}, 'count': 1}],
                'organismAgeUnit': [{'term': None, 'count': 1}],
                'organismAgeValue': [{'term': None, 'count': 1}],
            }
```

Note that this subtest fails currently with the response values of `organismAge` being `[{'term': {'value': 'None', 'unit': 'None'}, 'count': 1}]` (None is returned as the string 'None' instead of a null value)